deploy/localtest/slow-query-processor.yaml

```diff
@@ -0,0 +1,16 @@
+filebeat.inputs:
+  - type: stdin
+    multiline.type: pattern
+    multiline.pattern: '^# Time: '
```

Comment:
It might be a good idea not to rely on `Time` since the field order can be changed.

Reply:
I think we can only rely on the order of each line at the moment. Filebeat's multiline can only be used in this way.